jpa/bin/pom.xml

```diff
@@ -65,27 +79,51 @@
             <version>${junit.jupiter.version}</version>
         </dependency>
         <dependency>
-            <groupId>jakarta.tck</groupId>
+            <groupId>${project.groupId}</groupId>
             <artifactId>${tck.common.artifactId}</artifactId>
-            <version>${version}</version>
+            <version>${project.version}</version>
             <scope>test</scope>
+            <exclusions>
+                <exclusion>
+                    <groupId>jakarta.persistence</groupId>
+                    <artifactId>jakarta.persistence-api</artifactId>
+                </exclusion>
+            </exclusions>
         </dependency>
         <dependency>
-            <groupId>jakarta.tck</groupId>
+            <groupId>jakarta.persistence</groupId>
+            <artifactId>jakarta.persistence-api</artifactId>
+            <version>3.2.0-M2</version>
```

Comment:
this should be a property, since one thing is to test provider runtime, the other is to test API jar - both should be supported